packages/smooth_app/lib/pages/folksonomy/folksonomy_card.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:provider/provider.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/helpers/product_cards_helper.dart';
+import 'package:smooth_app/pages/folksonomy/folksonomy_page.dart';
+import 'package:smooth_app/pages/folksonomy/folksonomy_provider.dart';
+import 'package:smooth_app/pages/folksonomy/tag.dart';
+import 'package:smooth_app/pages/product/common/product_refresher.dart';
+import 'package:smooth_app/themes/constant_icons.dart';
+
+class FolksonomyCard extends StatelessWidget {
+  const FolksonomyCard(this.product);
+  final Product product;
+
+  @override
+  Widget build(BuildContext context) {
+    return ChangeNotifierProvider<FolksonomyProvider>(
+      create: (_) => FolksonomyProvider(product.barcode!),
+      child: Card(product),
+    );
+  }
+}
+
+class Card extends StatelessWidget {
```

Comment:
Could you give a more relevant name?
`Card` is a Widget from the Flutter framework.

Also, if this Widget is not used outside, please make it private.